docs/data-model.rst

```diff
@@ -117,15 +119,14 @@ Network
 
   !define SHOW_NETWORK
   !define SHOW_NETWORK_INTERNAL
+  !define SHOW_NETWORK_GENEALOGY
```

Comment:
I think we can just group `SHOW_NETWORK_GENEALOGY` under `SHOW_NETWORK_INTERNAL`